src/components/common/Dialogs/RemoveSafeDialog/RemoveSafeDialogForm.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { useFormContext } from 'react-hook-form';
+
+import Button from '~shared/Button';
+import DialogSection from '~shared/Dialog/DialogSection';
```

Comment:
You can also export this section like this `import { DialogSection } from '~shared/Dialog';`, that way we avoid repeating the `DialogSection` at the end of the import route